src/components/Pages/Search/Search.js

```diff
@@ -1,7 +1,102 @@
+import {
+  TextField,
+  ThemeProvider,
+  Button,
+  createMuiTheme,
+  Tabs,
+  Tab,
+} from "@material-ui/core";
+import { useState, useEffect } from "react";
+import SearchIcon from "@material-ui/icons/Search";
+import CustomPagination from "../../Pagination/CustomPagination";
+import SingleContent from "../../SingleContent/SingleContent";
+import "./Search.css";
+
 const Search = () => {
+  const [type, setType] = useState(0);
+  const [page, setPage] = useState(1);
+  const [searchText, setSearchText] = useState("");
+  const [content, setContent] = useState();
+  const [numOfPages, setNumOfPages] = useState();
+
+  const darkTheme = createMuiTheme({
+    palette: {
+      type: "dark",
+      primary: {
+        main: "#fff",
+      },
+    },
+  });
```

Comment:
This should move together with the `ThemeProvider`

Reply:
It's already in the App.js file.